src/main/webapp/app/iris/manage/settings/shared/iris-enabled/iris-enabled.component.html

```diff
@@ -1,15 +1,22 @@
-<div class="btn-group">
+<div class="btn-group" role="group" aria-label="Enable/Disable setting">
     <div
         class="btn"
-        [ngClass]="{ 'btn-success selected': irisSubSettings?.enabled, 'btn-default': !irisSubSettings?.enabled }"
-        [class.disabled]="disabled"
+        [ngClass]="{ 'btn-success selected': irisSubSettings?.enabled && !someButNotAllSettingsEnabled, 'btn-default': !irisSubSettings?.enabled || someButNotAllSettingsEnabled }"
```

Comment:
I think we should compute `irisSubSettings?.enabled && !someButNotAllSettingsEnabled` in the ts, we repeat this check in the HTML and it might lead to bugs in the future if adjustments are made and some parts of the HTML are forgotten 😅 

Also it makes the HTML easier to read